### ISSUE TITLE
Some spelling dictionaries (like Italian or Portuguese) will, when enabled, freeze wm

### ISSUE THREAD

**Issue by writemonkey** [writemonkey/wm3#45]

**Comment by RSchiaffino** [comment]
I confirm that the bug is still present: I've just downloaded the Italian .dic and .aff files, and, as soon as I try to switch spelling to Italian, WM freezes and needs to be closed via Task Manager.
Windows 10 64 bit

**Comment by RSchiaffino** [comment]
I would suggest that until this bug is resolved, the documentation should indicate not to try to use WM with the affected language(s)

**Comment by writemonkey** [comment]
Yes sorry for that. There is a warning in the docu, maybe not noticeable enough
https://github.com/writemonkey/wm3/wiki/Documentation#adding-dictionaries

I know this must be dealt with asap but I haven't find the solution for it yet. :(
i.

**Comment by guilherme-vairo** [comment]
Having this issue right now. Any update on a solution?
It unfreezes after a while, at least for Brazillian and European Portuguese.
